src/net/sourceforge/kolmafia/preferences/Preferences.java

```diff
@@ -531,12 +531,16 @@ private static void loadGlobalPreferences() {
   }
 
   private static void loadUserPreferences(final String username) {
+    String lusername;
```

Comment:
Instead of introducing a new variable, why not just remove the final qualifier from username?

Reply:
I had no idea why someone had declared it static so it was easier to leave it that way.  Habit from pre-Java where call by reference and call by value were considerations.  I'll change.